src/main/java/com/ncedu/fooddelivery/api/v1/entities/User.java

```diff
@@ -51,6 +51,10 @@
     @PrimaryKeyJoinColumn
     private Client client;
 
+    @OneToOne(cascade=CascadeType.ALL, mappedBy = "user")
+    @PrimaryKeyJoinColumn
+    private Courier courier;
+
```

Comment:
Сюда еще setter нужно запихнуть немного кастомный, чтобы работало создание объекта User и Courier/Moderator/Client просто через сущность User (я рассказывал эту проблему на позапрошлом митинге кажется).
 ```suggestion
public void setCourier(Courier courier) {
        this.courier = courier;
        this.courier.setUser(this);
    }
```